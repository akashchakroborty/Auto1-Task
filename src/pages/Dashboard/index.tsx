import React from 'react';
import { makeStyles } from '@mui/styles';
import { Container, Grid, Paper } from '@mui/material';
import CarList from '../../components/CarList/CarList.container';
import Filters from '../../components/Filters/Filters.container';

const useStyles = makeStyles({
  container: {
    marginTop: 24,
  },
});

const Dashboard: React.FC = () => {
  const classes = useStyles();
  return (
    <Container className={classes.container}>
      <Grid container spacing={2}>
        <Grid item xs={12} sm={12} md={4}>
          <Paper variant="outlined" square>
            <Filters />
          </Paper>
        </Grid>
        <Grid item xs={12} sm={12} md={8}>
          <CarList />
        </Grid>
      </Grid>
    </Container>
  );
};

export default Dashboard;
