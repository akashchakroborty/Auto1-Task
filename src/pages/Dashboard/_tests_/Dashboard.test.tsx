import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from '..';

describe('should test the Dashboard Component', () => {
  test('should render the component', () => {
    const component = shallow(<Dashboard />);
    expect(component).toMatchSnapshot();
  });
});
