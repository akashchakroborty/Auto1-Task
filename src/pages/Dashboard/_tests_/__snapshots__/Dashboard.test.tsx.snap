// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the Dashboard Component should render the component 1`] = `
<ForwardRef(Container)
  className="makeStyles-container-1"
>
  <ForwardRef(Grid)
    container={true}
    spacing={2}
  >
    <ForwardRef(Grid)
      item={true}
      md={4}
      sm={12}
      xs={12}
    >
      <ForwardRef(Paper)
        square={true}
        variant="outlined"
      >
        <Connect(Filters) />
      </ForwardRef(Paper)>
    </ForwardRef(Grid)>
    <ForwardRef(Grid)
      item={true}
      md={8}
      sm={12}
      xs={12}
    >
      <Connect(CarList) />
    </ForwardRef(Grid)>
  </ForwardRef(Grid)>
</ForwardRef(Container)>
`;
